cypress/support/step-definitions/numeral-date-steps.js

```diff
@@ -0,0 +1,58 @@
+import { numeralDateInputByPosition, numeralDateInput, numeralDateComponent } from '../../locators/numeralDate';
+import { ERROR_TOOLTIP } from '../../locators/form/locators';
+
+const BORDER_RED_COLOR = 'rgb(199, 56, 79)';
+const FIRST_INPUT = 0;
+const SECOND_INPUT = 1;
+const THIRD_INPUT = 2;
+
+Then('Number Date component has {int} separate inputs', (index) => {
+  for (let i = 0; i < index; ++i) {
+    numeralDateInputByPosition(i).should('be.visible');
+  }
+});
+
+Then('All {int} Numeral Date inputs have red border', (index) => {
```

Comment:
do we need this All here ? 
```suggestion
Then('{int} Numeral Date inputs have red border', (index) => {
```

Reply:
fixed